content/articles/how-to-implement-caching-in-adonisjs-5/index.md

```diff
@@ -0,0 +1,221 @@
+# How to Implement Caching in AdonisJS 5.
+
+The performance of a web application is becoming an integral part of the application as many statistics have shown that slow web applications can result in loss of traffic, sales, and incomes in many different ways.
+
+In this article, we will learn how to implement a Caching System on our Forum API developed with AdonisJS and integrate the newly created Caching Package for the new AdonisJS 5 with optimum Caching experience.
```

Comment:
Use consistent capitalization. eg. The words "Caching" and "System" should be lower case. Only capitalize nouns. Please revise your document.